__std__/flake.nix

```diff
@@ -26,7 +26,7 @@
       url = "github:input-output-hk/haskell.nix/e36a34c4abc02287ec79bf88a0c993f6b66f9945";
     };
     hackage-nix = {
-      url = "github:input-output-hk/hackage.nix/d2077b0703f10ab08a071669729805d3bfea340d";
+      url = "github:input-output-hk/hackage.nix/18e082f5e8aa744e26a159dd31477222aa298380";
```

Comment:
why not just remove the revision

Reply:
Yes I suppose there is no reason to keep them, as they can be found in `flake.lock` anyway.